lib/miq_automation_engine/service_models/miq_ae_service_service.rb

```diff
@@ -48,6 +48,30 @@ def set_dialog_option(key, value)
       end
     end
 
+    def service_vars_options
+      @object.options[:service_vars] || HashWithIndifferentAccess.new
+    end
+
+    def get_service_vars_option(key)
+      service_vars_options[key]
+    end
+
+    def set_service_vars_option(key, value)
+      ar_method do
+        @object.options[:service_vars] ||= HashWithIndifferentAccess.new
```

Comment:
Do we need to use `HashWithIndifferentAccess`?  I know we've had problems deserializing after upgrades before.  cc @jrafanie

Reply:
IIRC, the namespace of HWIA changed in a Rails version upgrade and we had to create an alias or something in order to deserialize them, but I can't find it right now so maybe I'm thinking of something else.